source/includes/environment_config/_25-show_with_remote.md.erb

```diff
@@ -0,0 +1,95 @@
+## Get environment with remote config
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/environments/my_environment/withremote' \
+      -u 'username:password' \
+      -H 'Accept: <%= data.apis.versions.environments %>'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: <%= data.apis.versions.environments %>; charset=utf-8
+ETag: "4167e3ec81fdac0fb29d854b36ceb981"
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "https://ci.example.com/go/api/admin/environments/my_environment"
+    },
+    "doc": {
+      "href": "https://api.gocd.io/#environment-config"
+    },
+    "find": {
+      "href": "https://ci.example.com/go/api/admin/environments/:environment_name"
+    }
+  },
+  "name": "my_environment",
+  "pipelines": [
+    {
+      "_links": {
+        "self": {
+          "href": "https://ci.example.com/go/api/admin/pipelines/up42"
+        },
+        "doc": {
+          "href": "https://api.gocd.io/#pipeline-config"
+        },
+        "find": {
+          "href": "https://ci.example.com/go/api/admin/pipelines/:pipeline_name"
+        }
+      },
+      "name": "up42"
+    }
+  ],
+  "agents": [
+    {
+      "_links": {
+        "self": {
+          "href": "https://ci.example.com/go/api/agents/adb9540a-b954-4571-9d9b-2f330739d4da"
+        },
+        "doc": {
+          "href": "https://api.gocd.io/#agents"
+        },
+        "find": {
+          "href": "https://ci.example.com/go/api/agents/:uuid"
+        }
+      },
+      "uuid": "12345678-e2f6-4c78-123456789012"
+    }
+  ],
+  "environment_variables": [
+    {
+      "secure": false,
+      "name": "username",
+      "value": "admin"
+    },
+    {
+      "secure": true,
+      "name": "password",
+      "encrypted_value": "LSd1TI0eLa+DjytHjj0qjA=="
+    }
+  ]
+}
+```
+
+Gets environment config along with the remote configurations for specified environment name.
+
+<aside class="notice">
+  <strong>Note:</strong> Please remember to save the value of the `ETag` header
```

Comment:
Is this required? There won't be any conditional posts as this is in config repo?